Packages/vcs/Lib/projection.py

```diff
@@ -21,6 +21,9 @@
 round_projections = ['polar (non gctp)', 'stereographic',
                      'orthographic', "ortho", ]
 
+no_over_proj4_parameter_projections = round_projections+["aeqd", "lambert conformal c"]
```

Comment:
In my original pull request I also did not pass +over for 'polar stereographic'. Maybe this is the reason why this you are failing some polar tests.